app/src/main/java/org/wikipedia/talk/UserTalkPopupHelper.kt

```diff
@@ -114,20 +115,25 @@ object UserTalkPopupHelper {
             override fun onMenuItemSelected(menu: MenuBuilder, item: MenuItem): Boolean {
                 when (item.itemId) {
                     R.id.menu_user_profile_page -> {
+                        PatrollerExperienceEvent.logAction("menu_user_page_click", "pt_diff_view")
```

Comment:
Same concerns as above: Should we only send the event when the action was from patroller edits?